tox.ini

```diff
@@ -30,35 +31,28 @@ passenv =
 description = Apply coding style standards to code
 deps =
     black
-    ruff
 commands =
-    ruff --fix {[vars]all_path}
     black {[vars]all_path}
 
 [testenv:lint]
 description = Check code against coding style standards
 deps =
     black
-    ruff
     codespell
 commands =
     codespell --skip .git --skip .tox --skip build --skip lib --skip venv --skip .mypy_cache {toxinidir}
-    ruff {[vars]all_path}
     black --check --diff {[vars]all_path}
 
-[testenv:static-{charm,lib}]
-description = Run static analysis checks
+[testenv:static-charm]
+description = Run static analysis checks for charm
 deps =
     pyright
-    charm: -r{toxinidir}/requirements.txt
-    lib: ops
-    lib: jinja2
-    unit: {[testenv:unit]deps}
-    integration: {[testenv:integration]deps}
+    ops
+    jinja2
 commands =
-    charm: pyright {[vars]src_path} {posargs}
-    lib: pyright --pythonversion 3.8 {[vars]src_path} {posargs}
-    lib: /usr/bin/env sh -c 'for m in $(git diff main --name-only {[vars]lib_path}); do if ! git diff main $m | grep -q "+LIBPATCH\|+LIBAPI"; then echo "You forgot to bump the version on $m!"; exit 1; fi; done'
+    pyright --pythonversion 3.6 {[vars]src_path} {posargs}
+    # FIXME: there's no LIB{PATCH,API} in src/charm.py, should there be?
```

Comment:
No, this would be in the charm libs but the charm doesn't have any. This can likely be removed